build/package/rpm/go-fdo-server.spec

```diff
@@ -14,19 +14,16 @@ Version:        0
 Release:        %autorelease -p
 Summary:        A Go implementation of the FIDO Device Onboard Specification
 
-# Generated by go-vendor-tools
 License:        Apache-2.0 AND BSD-3-Clause AND MIT
 URL:            %{gourl}
 Source0:        %{gosource}
-# Generated by go-vendor-tools
 Source1:        %{archivename}-vendor.tar.bz2
-Source2:        go-vendor-tools.toml
```

Comment:
Although we are not going to use the `go-vendor-tools.toml` file from within the spec file, it might worth it to maintain it in the sources so it's possible to generate the vendor tarball externally with `go-vendor-tools`. ??